external/rescue_proxy.go

```diff
@@ -114,6 +115,25 @@ func (c *RescueProxyAPIClient) GetWithdrawalAddresses() ([][]byte, error) {
 	return r.GetWithdrawalAddresses(), nil
 }
 
+func (c *RescueProxyAPIClient) ValidateEIP1271(dataHash *common.Hash, signature *[]byte, address *common.Address) (bool, error) {
+	// Connect if not yet connected.
+	if err := c.ensureConnection(); err != nil {
+		return false, err
+	}
+	c.logger.Debug("requesting eip1271 validation")
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+	r, err := c.client.ValidateEIP1271(ctx, &proxy.ValidateEIP1271Request{
+		DataHash:  dataHash.Bytes(),
+		Signature: *signature,
+		Address:   address.Bytes(),
+	})
+	if err != nil {
+		return false, err
+	}
+	return r.GetValid(), nil
```

Comment:
presumably if `r.Error` is set here, you want to return an error in the http response, but right now we just return 'false, nil'

Reply:
Done, pushed

Reply:
Wait I think I misread this comment last night...

Reply:
Ok pushed another commit. Is this what you were suggesting?

Reply:
looks good to me.